checklists/alz_checklist.en.json

```diff
@@ -557,7 +549,7 @@
         {
             "category": "Resource Organization",
             "subcategory": "Regions",
-            "text": "Consider a multi-region deployment. Depending on customer size, locations, and users presence, operating in multiple regions can be a common choice to deliver services and run applications closer to them. Using a multi-region deployment is also important to provide geo disaster recovery capabilities, to eliminate the dependency from a single region capacity and diminish the risk of a temporary and localized resource capacity constraint",
+            "text": "Deploy your Azure landing zone in a multi-region deployment. Depending on customer size, locations, and users presence, operating in multiple regions can be a common choice to deliver services and run applications closer to them. Using a multi-region deployment is also important to provide geo disaster recovery capabilities, to eliminate the dependency from a single region capacity and diminish the risk of a temporary and localized resource capacity constraint",
```

Comment:
Not all customers will have requirements / needs for multi region?

Reply:
Where I think we have landed with updating the CAF documentation and landing zone is that multi region *should* be the goal for customers; they need it for DR at the very least.  If the customer opts not to, that is of course fine - but as a best practice, having the landing zone ready to support the scenarios should be done.  The diagrams have been updated to represent that: https://aka.ms/alz/resiliency/blog

I don't think all customers will use it, just like all customers don't use policy, or cost management, but I think it should be something that customers understand they should use.